packages/devtools_app/lib/src/standalone_ui/vs_code/debug_sessions.dart

```diff
@@ -24,54 +24,41 @@ class DebugSessions extends StatelessWidget {
       children: [
         Text(
           'Debug Sessions',
-          style: Theme.of(context).textTheme.titleSmall,
+          style: Theme.of(context).textTheme.titleMedium,
         ),
         if (sessions.isEmpty)
           const Text('Begin a debug session to use DevTools.')
         else
           Table(
             columnWidths: const {
-              0: FlexColumnWidth(2),
-              // TODO(dantup): Fixed width icons+menu?
-              1: FlexColumnWidth(),
+              0: FlexColumnWidth(),
             },
+            defaultColumnWidth:
+                FixedColumnWidth(actionsIconSize + denseSpacing),
             defaultVerticalAlignment: TableCellVerticalAlignment.middle,
             children: [
               for (final session in sessions)
-                TableRow(
-                  children: [
-                    Text(
-                      '${session.name} (${session.flutterMode})',
-                      style: Theme.of(context).textTheme.titleSmall,
-                    ),
-                    if (api.capabilities.openDevToolsPage)
-                      _DevToolsMenu(api: api, session: session),
-                  ],
-                ),
+                _debugSessionRow(session, context),
             ],
           ),
       ],
     );
   }
-}
-
-class _DevToolsMenu extends StatelessWidget {
-  const _DevToolsMenu({required this.api, required this.session});
-
-  final VsCodeApi api;
-  final VsCodeDebugSession session;
 
-  @override
-  Widget build(BuildContext context) {
+  TableRow _debugSessionRow(VsCodeDebugSession session, BuildContext context) {
     // TODO(dantup): What to show if mode is unknown (null)?
     final mode = session.flutterMode;
     final isDebug = mode == 'debug';
     final isProfile = mode == 'profile';
     // final isRelease = mode == 'release' || mode == 'jit_release';
     final isFlutter = session.debuggerType?.contains('Flutter') ?? false;
 
-    return MenuBar(
+    return TableRow(
       children: [
+        Text(
+          '${session.name} (${session.flutterMode})',
```

Comment:
I dropped the mode if it's null. I think if we want to show explicitly that it's starting, we should tweak the API to make a more explicit status rather than using this field to infer it (I think they're equivalent but it feels like a slightly weird assumption).